pkg/agent/client.go

```diff
@@ -248,6 +248,16 @@ func (a *AgentClient) initializeAuthContext(ctx context.Context) (context.Contex
 // The requests include things like opening a connection to a server,
 // streaming data and close the connection.
 func (a *AgentClient) Serve() {
+	defer func() {
+		// close all of conns with remote when AgentClient closed
+		a.connManager.mu.Lock()
+		klog.V(2).InfoS("cleanup all of conn contexts when agent client exits", "agentID", a.agentID)
+		for _, ctx := range a.connManager.connections {
```

Comment:
To avoid confusion can we change ctx to something else (maybe conn or connCtx)? We generally reserve ctx for variables of type context.Context which this isn't.

Reply:
yeah,  changed ctx to connCtx